### PR TITLE
[docs] getting started and upgrade

### PR DESCRIPTION
**Pull request by jamiltz**
This change is [Reviewable](https://reviewable.io/reviews/couchbase/couchbase-lite-net/1015)

### CODE REVIEW

**Comment by jamiltz** [comment]
Before merging this PR, would it possible to fill in the [sample code](https://github.com/couchbase/couchbase-lite-net/pull/1015/files#diff-7418a0bfd9facad15b7f1fcda4e74ad8)? The swift version can be found [here](https://github.com/couchbase/couchbase-lite-ios/blob/docs-getting-started/Swift/Tests/SampleCodeTest.swift#L554-L625) and [here](https://github.com/couchbase/couchbase-lite-ios/blob/docs-getting-started/Swift/Tests/SampleCodeTest.swift#L144-L148) respectively.

File: docs/modules/ROOT/pages/csharp.adoc [hunk 1]
```diff
@@ -6,21 +6,41 @@ Couchbase
 Getting Started
 ~~~~~~~~~~~~~~~
 
-* Add `http://mobile.nuget.couchbase.com/nuget/Developer/` to your Nuget
-package sources and expect a new build approximately every 2 weeks!
+UWP project
+^^^^^^^^^^^
+
+Create or open an existing Virtual Studio project and install Couchbase Lite using the following method.
+
```
Comment (borrrden) [inline docs/modules/ROOT/pages/csharp.adoc]: What is the above section about?

File: docs/modules/ROOT/pages/csharp.adoc [hunk 2]
```diff
@@ -6,21 +6,41 @@ Couchbase
...
+class that is included in the support assembly (there is only one public class in each support assembly, and the support assembly itself is a nuget dependency).
+
+For example, UWP looks like `Couchbase.Lite.Support.UWP.Activate()`. Currently the support assemblies provide dependency injected mechanisms for default directory logic, and platform specific logging (i.e. Android will log to logcat with correct log levels and tags. No more "mono-stdout" at always info
 level).
 
+Starter code
+^^^^^^^^^^^^
+
+Open *Main.cs* in Xcode and copy the following code in the `main` method. This snippet demonstrates
+how to run basic CRUD operations, a simple Query and running
```
Comment (borrrden) [inline docs/modules/ROOT/pages/csharp.adoc]: Not Xcode

**Comment by borrrden** [comment]
Also there is a swift typo -> https://github.com/couchbase/couchbase-lite-ios/blob/docs-getting-started/Swift/Tests/SampleCodeTest.swift#L625

**Comment by borrrden** [comment]
The getting started is just a combination of all the other existing snippets repeated?

File: docs/modules/ROOT/pages/csharp.adoc (hunk 1 above)
Comment (jamiltz) [reply docs/modules/ROOT/pages/csharp.adoc]: It's the equivalent to the 1.x getting started https://developer.couchbase.com/documentation/mobile/1.5/installation/net/index.html

**Comment by jamiltz** [comment]
Well, the goal is to get people comfortable with the minimum amount of code, a bit like the 1.x version https://developer.couchbase.com/documentation/mobile/1.5/installation/net/index.html#getting-started

The version on this PR (https://github.com/couchbase/couchbase-lite-net/blob/doc-updates/docs/modules/ROOT/pages/csharp.adoc#getting-started) is a bit raw. Let's keep this PR open, I'll push further updates.

**Comment by jamiltz** [comment]
Oops, didn't intend to close the PR.

**Comment by jamiltz** [comment]
Updated. I think this PR can be merged unless there are other pending changes.

**Comment by borrrden** [comment]
If you don't mind my latest commit, go ahead and merge

**Comment by jamiltz** [comment]
Ok. Would you rather have the commits on this branch squashed or not before merging? I'm not sure what the usual process is for that.

**Pull request merged**